forage-android/src/main/java/com/joinforage/forage/android/ForageSDKInterface.kt

```diff
@@ -62,10 +62,25 @@ internal interface ForageSDKInterface {
      * payment capture. On success, returns a confirmation of the transaction.
      * On failure, provides a detailed error response.
      *
-     * @throws ForageConfigNotSetException If the passed ForagePANEditText instance
+     * @throws ForageConfigNotSetException If the passed ForagePINEditText instance
      * hasn't had its ForageConfig set via .setForageConfig().
      */
     suspend fun capturePayment(params: CapturePaymentParams): ForageApiResponse<String>
+
+    /**
+     * Collects the pin for a Forage Payment associated with an EBT card
```

Comment:
See suggestions from above
```suggestion
     * Collect a customer's PIN for an EBT payment and defer the capture of the payment to the server.
```